pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -10,95 +24,613 @@
 
 from pandas._libs import lib
 from pandas._libs.tslibs import (
-    NaT,
+    OutOfBoundsTimedelta,
     iNaT,
 )
 
 import pandas as pd
 from pandas import (
+    NA,
+    NaT,
     Timedelta,
     TimedeltaIndex,
     offsets,
     to_timedelta,
 )
 import pandas._testing as tm
 
+TD_UNITS = (
```

Comment:
Some, but not all, of these seem to be defined in other modules, though they're not always exposed for testing, e.g. https://github.com/pandas-dev/pandas/blob/1bc27414c36b185861c8d15fc44064db08feef72/pandas/_libs/tslibs/timedeltas.pyx#L95